bench-scripts/apache_bench.sh

```diff
@@ -53,20 +53,14 @@ set -x
 #
 
 INSTALL_ROOT=${BENCH_INSTALL_ROOT:-"/tmp/bench.binaries"}
-RESULT_DIR=${BENCH_RESULTS:-"${INSTALL_ROOT}/results"}
 WORKSPACE_ROOT=${BENCH_WORKSPACE_ROOT:-"/tmp/bench.workspace"}
 MAKE_OPTS=${BENCH_MAKE_OPTS}
 HTTPS_PORT=${BENCH_HTTPS_PORT:-'4430'}
 HTTP_PORT=${BENCH_HTTP_PORT:-'8080'}
-CERT_SUBJ=${BENCH_CERT_SUBJ:-'/CN=localhost'}
```

Comment:
I think those should stay here. The thing is we immediately see what is meant to be adjusted by opening the script we are going to execute. let's keep it until we find a better way to document things.